src/args.rs

```diff
@@ -136,6 +136,9 @@ Less common options:
     -L, --follow
         Follow symlinks.
 
+    --maxdepth NUM
+        Descend at most NUM directories below the command line arguments.
+
```

Comment:
Could you add this flag to `doc/rg.1.md` and run the convert script to create a new `doc/rg.1` file?